tests/cli/test_boot_tar_kickstart.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Note: execute this file from the project root directory
+
+#####
+#
+# Test the liveimg installed tar disk image
+#
+#####
+
+set -e
+
+. /usr/share/beakerlib/beakerlib.sh
+. $(dirname $0)/lib/lib.sh
+
+rlJournalStart
+    rlPhaseStartTest "Verify VM instance"
+        # Just the fact that this is running means the image can boot and ssh is working
```

Comment:
I prefer having the same asserts before and after the change so we can still make use of the helper functions in more importantly extend the asserts in the future with validations that apply to all image types.

The assert you currently have  will become a bit redundant b/c if we can ssh into localhost using the existing keys there's not much point in checking if `authorized_keys` exists.

Adding `CHECK_CMDLINE=0 verify_image root localhost "-p 22` should work here, see https://github.com/weldr/lorax/pull/885/commits/5af8fb1ef99cbb70b54b21bd4ace96d0f20e2059. Also note the part where we upload the existing keys to the VM in `check-cli:L45` in above commit.

Reply:
Cherry picked the docker->podman commit, 5af8fb, and added the changes to the new boot script as part of that commit.